src/hosts.ts

```diff
@@ -0,0 +1,33 @@
+import Registry from './registry';
+import parseURL from './parse-url';
+
+/**
+ * Hosts
+ *
+ * a map of hosts to Registries, ultimately allowing
+ * a per-host-and-port, per HTTP verb lookup of RouteRecognizers
+ */
+export default class Hosts {
+
+  private registries = {};
```

Comment:
Build outputs would be `Hosts.registries` rather than `Hosts._registries` before. Not sure whether it should be considered as breaking change.